packages/core-app-api/src/apis/implementations/FetchApi/PluginProtocolResolverFetchMiddleware.test.ts

```diff
@@ -106,14 +100,14 @@ describe('PluginProtocolResolverFetchMiddleware', () => {
     expect(inner.mock.calls[0][0]).toBe('https://elsewhere.com');
     expect(inner.mock.calls[0][1].body).toBe('123');
 
-    await outer(
-      new Request('plugin://a', {
-        method: 'POST',
-        body: '123',
-      }),
-    );
+    const inputRequest = new Request('plugin://a', {
+      method: 'POST',
+      body: '123',
+    });
+
+    await outer(inputRequest);
 
     expect(inner.mock.calls[1][0]).toBe('https://elsewhere.com');
-    expect(inner.mock.calls[1][1].body).toEqual(Buffer.from('123', 'utf8'));
+    expect(inner.mock.calls[1][1]).toBe(inputRequest);
```

Comment:
Unsure if this is an okay fix or not. The issue is, the `inner.mock.calls[1][1].body` is now a `ReadableStream`, and I also think the internal `new Request(input ...)` in the `apply` method consumes this stream.